frontend/src/pages/Admin/Components/Resource/resource.module.scss

```diff
@@ -0,0 +1,93 @@
+.resource-card-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+  gap: 10px;
+  padding: 1.5rem;
+}
+.resource-card {
+  padding: 2rem;
+  background: linear-gradient(45deg, rgb(113, 113, 245), rgb(71, 71, 219));
+  border-radius: 10px;
+}
+
+.resource-card:hover {
+  box-shadow: 0px 2px 2px 2px gray;
+}
+
+.resource-card > h2 {
+  text-align: center;
+  color: #fff;
+}
+.resource-card > h4 {
+  text-align: center;
+  color: #fff;
+  margin: 10px;
+}
+.button-group {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+.button-url {
+  padding: 10px 30px;
+  border: none;
+  outline: none;
+  border-radius: 5px;
+  background: linear-gradient(45deg, rgb(115, 196, 228), rgb(111, 111, 247));
+  margin: 5px;
+  color: #fff;
+  width: 120px;
+  font-weight: bold;
+}
+.button-url:hover {
+  background: linear-gradient(45deg, rgb(111, 111, 247), rgb(115, 196, 228));
+}
+.button-delete {
+  padding: 10px 30px;
+  border: none;
+  outline: none;
+  border-radius: 5px;
+  background: linear-gradient(45deg, rgb(255, 0, 0), rgb(243, 109, 109));
+  margin: 5px;
+  color: #fff;
+  width: 120px;
+  font-weight: bold;
+}
+.button-delete:hover {
+  background: linear-gradient(45deg, rgb(243, 109, 109), rgb(255, 0, 0));
+}
+.description-card {
+  padding: 2rem;
+  margin: 0;
+  font-size: 1.5rem;
+  color: #fff;
+}
+.description-card > p {
+  margin: 10px;
+}
+.additional-info {
+  font-size: 1rem;
+}
+.expire-update-cards {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  text-align: center;
+  color: rgb(41, 41, 41);
+}
+.expire-update-card {
+  background: linear-gradient(45deg, rgb(144, 144, 236), rgb(165, 178, 255));
```

Comment:
Please make the background-color same as the broadcasts page in order to maintain the consistency.